### PR TITLE
Documenting the icon colors used (for creating new quest icons)

### PR DESCRIPTION
**Pull request by mnalis**
I did initial cataloguing going over https://wiki.openstreetmap.org/wiki/StreetComplete/Quests

Needs more work and more general descriptions, see `FIXME` section and `?` in entries that indicate strangeness/inconsistencies in coloring.

If someone could shed a light on those, it would be great.

### CODE REVIEW

File: CONTRIBUTING_A_NEW_QUEST.md [hunk 1]
```diff
@@ -317,7 +317,20 @@ Note that there are some graphics which haven't been used yet, created for propo
...
+Keep similar style to existing ones and app in general. Note the background color of the icon marks its relation group:
+- magenta: bicycle traffic
+- blue: pedestrian traffic
+- green: pedestrian misc (nature, picnic, sport, religion, recycling, police, postbox, wheelchair, summits, AED, toilets, backrest ...)
+- yellow: motor vehicles (car, motorcycles)
+
+FIXME:
+- gold: stile type/steps, trees/orchads
+- peach: survelience(?) and shop/money related (laundry, opening hours, shop types/seating, shop/atm names, shop level, aircondition, smoking, wlan, bicycle repair/shops, food, payment) 
+- grey: building type/height/entrances/roof/address, power poles, briges, fire hydrants
```
Comment (matkoniecz) [inline CONTRIBUTING_A_NEW_QUEST.md:329]: that seems match to "man made constructions"

File: CONTRIBUTING_A_NEW_QUEST.md [hunk 2]
```diff
@@ -317,7 +317,20 @@ Note that there are some graphics which haven't been used yet, created for propo
...
+
+- why is "Do you need to pay to enter here?" green instead of peach (it is about money)
```
Comment (matkoniecz) [inline CONTRIBUTING_A_NEW_QUEST.md:331]: "Is cash payment accepted here?" is orange already

File: CONTRIBUTING_A_NEW_QUEST.md [hunk 3]
```diff
@@ -317,7 +317,20 @@ Note that there are some graphics which haven't been used yet, created for propo
...
 
-Keep similar style to existing ones and app in general. Once the quest icon is ready:
+Keep similar style to existing ones and app in general. Note the background color of the icon marks its relation group:
+- magenta: bicycle traffic
+- blue: pedestrian traffic
+- green: pedestrian misc (nature, picnic, sport, religion, recycling, police, postbox, wheelchair, summits, AED, toilets, backrest ...)
+- yellow: motor vehicles (car, motorcycles)
+
+FIXME:
+- gold: stile type/steps, trees/orchads
```
Comment (matkoniecz) [inline CONTRIBUTING_A_NEW_QUEST.md:327]: for stile steps it prevents duplicate icon with steps count within regular steps

File: CONTRIBUTING_A_NEW_QUEST.md (hunk 2 above)
Comment (westnordost) [reply CONTRIBUTING_A_NEW_QUEST.md:331]: Because it is about amenities (museum, art galleries, ...) and not shops.

**Review by westnordost — COMMENTED** [review]
green is "amenities",
peach is "shop" things
grey is "man made / construction"

**Comment by matkoniecz** [comment]
I guess that camping quests triggered this - and conclusion seems that green is correct there. One thing open is - why brown? It seems to be used for nature-related questions.

**Comment by westnordost** [comment]
> It seems to be used for nature-related questions.

correct

**Review by matkoniecz — APPROVED** [review]

File: CONTRIBUTING_A_NEW_QUEST.md [hunk 4]
```diff
@@ -317,7 +317,16 @@ Note that there are some graphics which haven't been used yet, created for propo
 
 A new icon can reuse the content of [other quest icons](res/graphics/quest), it can be based on openly licensed graphics such as ones from [svgrepo.com](https://www.svgrepo.com/). See [the attribution file](res/graphics/authors.txt) for what was used so far.
 
-Keep similar style to existing ones and app in general. Once the quest icon is ready:
+Keep similar style to existing ones and app in general. Note the background color of the icon marks its relation group:
+- magenta: bicycle traffic
+- blue: pedestrian traffic
+- yellow: motor vehicles (car, motorcycles)
+- grey: contructions (building type/height/entrances/roof/address, power poles, bridges, fire hydrants...)
+- peach: shop related (opening hours, shop types/seating, shop/atm names, shop level, airconditioning, smoking, internet access, payment, survelience)
```
Comment (westnordost) [inline CONTRIBUTING_A_NEW_QUEST.md:325]:
```suggestion
- light orange: shop related (opening hours, shop types/seating, shop/atm names, shop level, airconditioning, smoking, internet access, payment, survelience) 
```

**Pull request merged**